xml/System.Runtime.InteropServices/NativeLibrary.xml

```diff
@@ -37,8 +37,8 @@
         <Parameter Name="handle" Type="System.IntPtr" />
       </Parameters>
       <Docs>
-        <param name="handle">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="handle">The native library handle to be freed.</param>
+        <summary>Frees a given a loaded library handle. No action if the input handle is <see langword="null" />.</summary>
```

Comment:
~~No action if the input handle is null~~ -> No action if the input handle is `IntPtr.Zero`

This should not be `null` since `IntPtr` is a value type.